### ISSUE TITLE
Accessibility Feedback - Design Intent - eBenefits [VR&E CH31]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#11434]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** Jennifer (VSA) & Trevor (VSP)

## Description

This ticket will house accessibility feedback for the VR&E CH31.

## Definition of done

1. Review and acknowledge feedback.
1. Document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions.

**Comment by jenstrickland** [comment]
1. For the wizard prototype, the clickable area **must** include both the radio input and the label.

1. The same treatment should apply for checkboxes, text inputs, dropdowns/selects — labels **must** be clickable and move focus to the related input.

**Comment by jenstrickland** [comment]
![Screen Shot 2020-07-20 at 3 55 18 PM](https://user-images.githubusercontent.com/57469/87980401-936a8b00-caa1-11ea-9c44-327b051913f7.png)

3. The accordion heading **should** not be repeated.

**Comment by jenstrickland** [comment]
![Screen Shot 2020-07-20 at 4 00 10 PM](https://user-images.githubusercontent.com/57469/87980849-34594600-caa2-11ea-98c3-282fac70439b.png)

4. The user should confirm their email address. In the BAM2 Medical Device Ordering Tool, there are two email inputs and they should match.

**Comment by jenstrickland** [comment]
![Screen Shot 2020-07-21 at 10 31 19 AM](https://user-images.githubusercontent.com/57469/88067927-90bd7380-cb3d-11ea-94b4-8ff99950e4bc.png)

5. The input fields **must** use the same patterns used elsewhere on VA.gov. [Cognition, consistency]

![Screen Shot 2020-07-21 at 10 31 01 AM](https://user-images.githubusercontent.com/57469/88068103-c6faf300-cb3d-11ea-89a9-3438e80e462d.png)

**Comment by jenstrickland** [comment]
6. Links **must** have 24 pixels space between. Additionally, please use a bulleted list for this information so the context is clear to sighted and screen reader users.

![Screen Shot 2020-07-21 at 11 05 27 AM](https://user-images.githubusercontent.com/57469/88071950-76d25f80-cb42-11ea-9951-37599a713939.png)

**Comment by jenstrickland** [comment]
7. For the wizard, at viewports less than 600 pixels wide, **consider** using the radio input buttons to provide a touch-friendly target. I believe this is the direction the wizard redesign is headed. 

![Screen Shot 2020-07-21 at 11 08 54 AM](https://user-images.githubusercontent.com/57469/88072159-b4cf8380-cb42-11ea-8671-73dcc12ade13.png)

For example, the buttons for smaller viewports might look like the following. 

![yes-no](https://user-images.githubusercontent.com/57469/88073809-9ec2c280-cb44-11ea-8b3b-816eec019098.png)

**Comment by jenstrickland** [comment]
8. To avoid confusion, **consider** placing names in Title Case rather than ALL CAPS. The ALL CAPS scenario only happens as a result of bad database formatting.


![Screen Shot 2020-07-21 at 1 41 41 PM](https://user-images.githubusercontent.com/57469/88088139-f585c780-cb57-11ea-87d5-c1968fb84cea.png)

**Comment by 1Copenut** [comment]
@jenstrickland I've reviewed your design feedback and this all looks 💯 

I have one question for item #3, and that's all from VSP. You stated
> In addition, be sure to indicate that focus is to move to the first heading within the accordion so that the user knows the accordion has opened and knows where they are if they can't see the screen or use a mouse.

Is this a new interaction pattern we're looking to adopt? Currently the review screen accordions don't move focus when users toggle them open or closed, relying on the `aria-expanded` attribute to let screen readers know an interaction occurred.

**Comment by jenstrickland** [comment]
> Is this a new interaction pattern we're looking to adopt? Currently the review screen accordions don't move focus when users toggle them open or closed, relying on the `aria-expanded` attribute to let screen readers know an interaction occurred.

This was the behavior that we documented in the fall when I reviewed the Higher-Level Review. 
1. User opens accordion
1. At present, focus stays on the button that opened the accordion, there is no indication that anything happened
1. The same interaction pattern happens when a user selects the Edit button to move into the inline editable content, and our recommendation was to move to the legend/heading within to communicate the context change
1. We should communicate a context change here

**Comment by jenstrickland** [comment]
Trevor and I discussed offline to resolve the question above. 

This feedback is ready for you, @jason-gcio and @sporeboy .

**Comment by jason-gcio** [comment]
@joshkimux - I am going to create a separate ticket to address this ticket as some or most of these have been addressed since this was written and I don't want to introduce new confusion.  This came to us a long time ago but I want to make sure I have left no stone unturned.  This is in addition to the orientation feedback you have recently helped us with.

For clarity, I am addressing:
1. through 4. would be removed as we have already done those, addressed them elsewhere
5. we still need to do
6. Not our page, but I can request from pubweb
7. We'll consider it but doubtful to be complete by launch
8. We cannot control data from CorpDB and are hesitant to manipulate data as it comes from a source of truth

We can consider closing once we have everything checked off.
